docs/develop/cron-jobs.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Cron Jobs
+---
+
+You can use a `CronJob` to run jobs on a time-based schedule. These automated jobs run similarly to [Cron](https://en.wikipedia.org/wiki/Cron) tasks.
+
+Cron jobs are useful for creating periodic and recurring tasks, like running a database query or sending emails. Cron jobs can also schedule individual tasks for a specific time, such as if you want to schedule a job for a low activity period.
+
+<Callout type="important">
+
+**Cron jobs** have limitations and idiosyncrasies:
+- In certain circumstances, a single cron job can create multiple jobs. Therefore, jobs should be **idempotent**.
```

Comment:
It's not clear what it means for a single job to create multiple jobs. How about:
```suggestion
- In certain circumstances, a single execution can result in your cron job endpoint being called more than once in order to provide at-least-once semantics. As a result jobs should be **idempotent**, meaning calling the cron job endpoint multiple times should behave the same way as calling it once.
```